compiler/damlc/daml-doc/test/DA/Daml/Doc/Render/Tests.hs

```diff
@@ -79,6 +79,18 @@ cases = [ ("Empty module",
                 (Just "function with context")
             ] [] []
           )
+        , ("Templates with interface implementations",
```

Comment:
Oh, isn't it possible to do a golden test in daml-test-files, like the others (e.g. `ConstraintTuples.EXPECTED.md`)? I guess it might not work because interfaces are only in 1.dev

Reply:
Ah, I wasn't aware of these golden tests. I deleted the ones I added here, and added golden tests for interface the daml-test-files.